src/Uno.UI/UI/Xaml/Controls/ScrollViewer/ScrollViewer.cs

```diff
@@ -969,16 +978,19 @@ void IFrameworkTemplatePoolAware.OnTemplateRecycled()
 		}
 
 		#region Content and TemplatedParent forwarding to the ScrollContentPresenter
-		protected override void OnContentChanged(object oldValue, object newValue)
+		protected override void OnContentChanged(object? oldValue, object? newValue)
 		{
-			base.OnContentChanged(oldValue, newValue);
-
-			if (_presenter != null)
+			if (oldValue is { } && oldValue != newValue)
```

Comment:
```suggestion
			if (oldValue is not null && oldValue != newValue)
```

You're so 2021 !

Reply:
We should also be comparing references, not using object.equals.